core/deployment/src/main/java/io/quarkus/deployment/pkg/builditem/DeploymentResultBuildItem.java

```diff
@@ -0,0 +1,26 @@
+
+package io.quarkus.deployment.pkg.builditem;
+
+import java.util.Map;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+public final class DeploymentResultBuildItem extends SimpleBuildItem {
+
+    private final String name;
+    private final Map<String, String> labels;
```

Comment:
Here https://github.com/quarkusio/quarkus/pull/5496/files#diff-5335230ed301829601e651decd6c5577R25 we pass `null` and `null`, so I am wondering if we need the two fields?

Reply:
The intention is to collect the name of the deployment and its labels and expose them as a build item, for anyone interested. This is not implemented yet.

Reply:
Thanks, for the explanation. I suppose I reviewed a little too early.